packages/client/hooks/useDraggableReflectionCard.tsx

```diff
@@ -337,6 +354,9 @@ const useCollapsePlaceholder = (
       // the card is the only one in the group, shrink the group!
       style.height = scrollHeight + 'px'
       style.transition = `height ${Times.REFLECTION_DROP_DURATION}ms`
+      const {remoteDrag} = reflection
+      // do not collapse if remote opened spotlight
+      if (remoteDrag?.isSpotlight) return
```

Comment:
Moved this here so that the `style.height` is the `scrollHeight`. Otherwise, when the Spotlight closes and the animating remote reflection is dropping into the kanban, the reflection below think the height is 0 and briefly move into its position.